.github/workflows/func.yaml

```diff
@@ -88,6 +88,10 @@ jobs:
           lxc profile set --project nested-lxd default security.privileged=true
           # set default project to be nested-lxd project
           juju model-default project=nested-lxd
+      - name: Show juju information
+        run: |
+          juju version
+          juju controllers
```

Comment:
Can we use only `juju show-controller | grep agent-version`, since printing all information about controller is not safe.

Reply:
Updated